src/FileConventions/Library.fs

```diff
@@ -329,3 +329,46 @@ let DetectInconsistentVersionsInFSharpScripts
         false
     else
         DetectInconsistentVersionsInNugetRefsInFSharpScripts fsxFiles
+
+let allowedNonVerboseFlags =
+    seq {
+        "unzip"
+
+        // even if env in linux has --split-string=foo as equivalent to env -S, it
+        // doesn't seem to be present in macOS' env man page and doesn't work either
+        "env -S"
+    }
+
+let NonVerboseFlagsInGitHubCI(fileInfo: FileInfo) =
```

Comment:
@realmarv this function should be renamed; if we're checking for non-verbose flags in .fsx files then it has nothing to do with GitHubCI

Reply:
Done